resources/systemsetup.rb

```diff
@@ -3,7 +3,7 @@
 resource_name :systemsetup
 
 property :setting, desired_state: false
-property :value, desired_state: true, coerce: proc { |m| m.to_s.capitalize }
+property :value, desired_state: true, coerce: proc { |m| m.to_s.capitalize unless m.to_s.include?('/') }
```

Comment:
Is this to fix the `set networktimeserver to Time.windows.com` behavior? I'm still seeing the capitalization in the Chef run, but the keep_awake smoke test doesn't validate it - is this because they're node attributes?